benchmark/src/cli_opt.rs

```diff
@@ -64,15 +53,34 @@ pub struct RubenOpt {
     /// A value of 1 ms effectively means starting all clients at once.
     #[structopt(short = "g", long = "stagger_range_ms", default_value = "64")]
     pub stagger_range_ms: u16,
+    /// Submit constant number of TXN requests per second; otherwise TXNs are flood to Libra.
+    #[structopt(short = "k", long = "submit_rate")]
+    pub submit_rate: Option<u64>,
```

Comment:
Is this common? Is it also used in throughput search?

Reply:
This is needed for creating Benchmarker, but it is not used in throughput test because we will control/change the submit rate.